content/2017/02/09/generating-documentation-polyglot-projects.rst

```diff
@@ -0,0 +1,123 @@
+public: yes
+author: david
+tags: [
+  Docs,
+  Tools,
+  Javascript,
+  Python,
+  Sass
+  ]
+summary: |
+  Code documentation is ideally written as close to the actual code
+  as possible, but compiled into a comprehensive set of documentation
+  that includes code from all languages in use. Here's how we intend
+  to do that.
+
+
+Generating code documentation for polyglot projects
+===================================================
+
+At OddBird we believe that undocumented code is unfinished code.
+The act of documenting clarifies what we are building,
+and the resulting documentation guides consistency in how we do things
+in different parts of a project built by different team members
+at different times.
+
+In a typical project, one of our documentation deliverables is a
+living styleguide [link?]
+which serves as documentation of the visual elements used in the app.
+The styleguide is built automatically whenever we build the code, and
+it is delivered as a static HTML website which can be served alongside the app.
+For example, the styleguide for this website can be found here:
+http://oddbird.net/styleguide/
+
```

Comment:
I'd change sentence to "For example, here's the style guide for OddSite." and make [style guide for OddSite] the linked text.